Makefile

```diff
@@ -4,6 +4,7 @@ help:
 	@echo "make update   -- upload the json and index.html to s3"
 
 generate:
+	wget https://hugovk.github.io/top-pypi-packages/top-pypi-packages-365-days.min.json -O data.json
```

Comment:
Looks very good. Just few questions here:
- Looks like we will have top packages since last year?
- How many packages does it include?

Reply:
Yes, it's get from the last 365 days. There's also an option for the last 30 days. Would you prefer that?

There's 5,000 packages listed in the file. It's about 240K in size. Actually, that file doesn't need to be committed to this repo, it's only needed as input to make results.json. Shall I remove `data.json` and ignore it?

Reply:
yea, 365 days is fine. In past, it was using even stats available for even longer time. But 365 days is probably better.

You are right, we don't need `data.json` file in the repo. Let's remove it & put on gitignore.

Reply:
Done!